karma.conf.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (config) => {
```

Comment:
Are you sure you need this config? I suppose it works with common config from core package

Reply:
Good point! seems it is same. Will remove it